### PR TITLE
TCD-2903: Backtrace - Unity

### PR DESCRIPTION
**Pull request by LoryApiFortress**
### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/error-reporting/platform-integrations/unity/help-shift.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
+---
+id: help-shift
+title: Unity Backtrace and HelpShift SDK integration
+sidebar_label: HelpShift SDK
+description: Integrate HelpShift SDK in Unity games.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/help-shift.md:5]:
```suggestion
description: Integrate HelpShift SDK in Unity games.
```

File: docs/error-reporting/platform-integrations/unity/help-shift.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+  - Refer to the [Unity documentation](https://docs.unity3d.com/Manual/webgl-interactingwithbrowserscripting.html) on how to embed a JavaScript library.
+  - Find the source code in our [example game](https://github.com/backtrace-labs/unity-asterax/blob/Helpshift-Webinar/Assets/Plugins/AsteraX.jslib)
+
+## 2. Linking HelpShift And Backtrace
+
+### iOS And Android
+
+For details on controlling the display of the HelpShift UI, refer to the [HelpShift SDK for Android documentation](https://developers.helpshift.com/sdkx-unity/support-tools-android/#conversation-view).
+
+Linking Backtrace and HelpShift is easy. Simply add the `backtraceClient["guid"]` to a customer issue field with the ID `device_id`, as shown in the code example below:
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/help-shift.md:39]:
```suggestion
Linking Backtrace and HelpShift is not tricky. Add the `backtraceClient["guid"]` to a customer issue field with the ID `device_id`, as shown in the code example below:
```

File: docs/error-reporting/platform-integrations/unity/metrics.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
+---
+id: metrics
+title: Stability Metrics Configuration
+sidebar_label: Stability Metrics Configuration
+description: Configure your Unity SDK for your project's stability metrics.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/metrics.md:5]:
```suggestion
description: Configure your Unity SDK for your project's stability metrics.
```

File: docs/error-reporting/platform-integrations/unity/metrics.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+For stability metrics to respond to queries and filters in the Web Console, follow these steps:
+
+1. Ensure that the SDK sends relevant attributes on events for the metric group in question.
+2. Create the metric group and associate the relevant attributes in the Web Console's **Stability Monitoring** project setting.
+
+By default, the Unity SDK sends events that contribute to the `Application Launches` metric group. This metric group calculates Crash Free Sessions and Crash Free Users on the [Overview dashboard](https://support.backtrace.io/hc/en-us/articles/360062537851-Overview-Page). This metric group serves as an example throughout this configuration process.
+
+### SDK Event Attributes
+
+Stability metrics like Crash Free Users/Sessions are calculated using events sent from the Unity SDK. To ensure that stability metrics respond to queries and filters in the Web Console (e.g., widgets on the Overview page), the attribute you want to use for filtering must be included in the events sent by the SDK. Refer to the following list of attributes that are sent by default with events from the Unity SDK:
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/metrics.md:37]:
```suggestion
Stability metrics like Crash Free Users/Sessions are calculated using events sent from the Unity SDK. To ensure that stability metrics respond to queries and filters in the Web Console (for example., widgets on the Overview page), the attribute you want to use for filtering must be included in the events sent by the SDK. Refer to the following list of attributes that are sent by default with events from the Unity SDK:
```

File: docs/error-reporting/platform-integrations/unity/metrics.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+Let's walk through an example to illustrate the setup process. Suppose you want to analyze crashes/errors against the minutes a user has played your game. This allows you to calculate metrics such as:
+
+- Average crash-free playtime for users
+- Average playtime per user
+- Average session length
+- Average session length by platform
+- Average session length by release version
+
+#### Step 1: Add the new metric group to your Backtrace Project in the Web Console by navigating to **Project Settings** > **Stability Monitoring**.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/metrics.md:128]:
```suggestion
#### Step 1: Add the new metric group to your Backtrace Project in the Web Console by navigating to **Project Settings** > **Stability Monitoring**
```

File: docs/error-reporting/platform-integrations/unity/metrics.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+
+In the above example, a new metric group called `MinutesPlayed` is created, linking the `application.version` and `uname.sysname` attributes to it.
+
+:::tip
+
+The **Metrics** section under **Attributes** needs to be left empty for Unity. This section applies only when importing external metrics.
+
+:::
+
+#### Step 2: Configure the Unity SDK to send this metric group and the desired attributes.
```
Comment (LoryApiFortress) [inline docs/error-reporting/platform-integrations/unity/metrics.md:140]:
```suggestion
#### Step 2: Configure the Unity SDK to send this metric group and the desired attributes
```

File: docs/error-reporting/platform-integrations/unity/help-shift.md [hunk 7]
```diff
@@ -0,0 +1,110 @@
+---
+id: help-shift
```
Comment (lysannep) [inline docs/error-reporting/platform-integrations/unity/help-shift.md:2]:
```suggestion
id: helpshift
```

File: docs/error-reporting/platform-integrations/unity/help-shift.md
Comment (lysannep) [inline docs/error-reporting/platform-integrations/unity/help-shift.md:1]: Can you rename the file from "help-shift.md" to "helpshift.md"?

File: sidebars.js [hunk 8]
```diff
@@ -398,6 +398,8 @@ module.exports = {
                                 'error-reporting/platform-integrations/unity/setup',
                                 'error-reporting/platform-integrations/unity/configuration',
                                 'error-reporting/platform-integrations/unity/attributes',
+                                'error-reporting/platform-integrations/unity/metrics',
+                                'error-reporting/platform-integrations/unity/help-shift',
```
Comment (lysannep) [inline sidebars.js:402]:
```suggestion
                                'error-reporting/platform-integrations/unity/helpshift',
```

**Review by lysannep — APPROVED** [review]

**Pull request merged**